unlock-app/src/components/interface/checkout/Connected.tsx

```diff
@@ -183,7 +183,7 @@ export function Connected({
   injectedProvider,
   children,
 }: ConnectedCheckoutProps) {
-  const [state, send] = useActor<CheckoutService>(service as CheckoutService)
+  const state = useSelector(service as CheckoutService, (state) => state)
```

Comment:
Would this syntax work? It's cleaner

```suggestion
  const state = useSelector<CheckoutService>(service, (state) => state)
```